src/components/cards/card.css

```diff
@@ -267,3 +267,6 @@
     background: $ui-white;
     box-shadow: 0px 0px 0px 2px $ui-black-transparent;
 }
+.tracking-pixel {
+    display: none;
+}
```

Comment:
Since some browsers won't load images if they have `display: none`, maybe we should just remove the `display: none` style, and just not deploy this until the images exist?